presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveCreateExternalTable.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import io.prestosql.tempto.AfterTestWithContext;
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.Requirement;
+import io.prestosql.tempto.RequirementsProvider;
+import io.prestosql.tempto.configuration.Configuration;
+import io.prestosql.tempto.hadoop.hdfs.HdfsClient;
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.Requirements.compose;
+import static io.prestosql.tempto.fulfillment.table.TableRequirements.mutableTable;
+import static io.prestosql.tempto.fulfillment.table.hive.tpch.TpchTableDefinitions.NATION;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static java.lang.String.format;
+
+public class TestHiveCreateExternalTable
+        extends ProductTest
+        implements RequirementsProvider
+{
+    @Inject
+    private HdfsClient hdfsClient;
+
+    @Inject
+    @Named("databases.hive.warehouse_directory_path")
+    private String warehouseDirectory;
+
+    @BeforeTestWithContext
+    public void setup()
+            throws Exception
+    {
+        hdfsClient.createDirectory(warehouseDirectory + "/TestHiveCreateExternalTable");
+    }
+
+    @AfterTestWithContext
+    public void cleanup()
+    {
+        hdfsClient.delete(warehouseDirectory + "/TestHiveCreateExternalTable");
+    }
```

Comment:
Use a shorter (not nested) path for `tableLocation` in the test code
and this will let you remove setup and cleanup from here

(even if you leave a table directory behind, it's not a problem)

Reply:
Done.